sql/core/benchmarks/AggregateBenchmark-results.txt

```diff
@@ -0,0 +1,154 @@
+================================================================================================
+aggregate without grouping
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
+Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
+
+agg w/o group:                           Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+agg w/o group wholestage off                39650 / 46049         52.9          18.9       1.0X
+agg w/o group wholestage on                   1224 / 1413       1713.5           0.6      32.4X
+
+
+================================================================================================
+stat functions
+================================================================================================
+
```

Comment:
@davies Do you know how to generate there benchmark:
https://github.com/apache/spark/blob/3c3eebc8734e36e61f4627e2c517fbbe342b3b42/sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/AggregateBenchmark.scala#L70-L78